scalaz-test/src/main/scala/ztest/IOSuite.scala

```diff
@@ -0,0 +1,67 @@
+package scalaz.test
```

Comment:
I'd prefer `scalaz.ztest`. Either that or drop `scalaz` and just go top-level for (almost) everything.